src/helpers/query/utils.js

```diff
@@ -74,6 +74,17 @@ export const getAxiosInstance = (db = 'admin', unauthenticated = false) => {
   const authStore = useAuthStore();
   const { roarfirekit } = storeToRefs(authStore);
   const axiosOptions = _get(roarfirekit.value.restConfig, db) ?? {};
+
+  // Add appCheckToken to the headers if it exists in the firekit config
+  const appCheckToken = roarfirekit.value[db]?.appCheckToken;
+
+  if (appCheckToken) {
+    axiosOptions.headers = {
+      ...axiosOptions.headers,
+      'X-Firebase-AppCheck': appCheckToken,
+    };
+  }
+
```

Comment:
Should we conditionally add this to the headers? Is there any harm in adding this if `appCheckToken` is `null` or `undefined`?

Reply:
Do you mean should we always attach the appCheckToken, even if it's null or undefined?

Otherwise, won't the `if (appCheckToken)` clause take care of this situation? Since that will be falsy if `appCheckToken `is null or undefined?

Reply:
OMG. Sorry. Please disregard. Apparently, I just can't read.